runtime/etc/Makefile.mli-static

```diff
@@ -21,7 +21,14 @@
 
 include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
 
-CHPL_MLI_INCS = -I$(CHPL_ZMQ_LOC)/include
+ifdef CHPL_ZMQ_HOME
+CHPL_MLI_INCS = -I$(CHPL_ZMQ_HOME)/include
+CHPL_ZMQ_LIB_HOME = $(CHPL_ZMQ_HOME)/lib
+CHPL_ZMQ_LIB = $(CHPL_ZMQ_LIB_HOME)/libzmq.a
+CHPL_ZMQ_DASH_L = -L$(CHPL_ZMQ_LIB_HOME)
+else
+CHPL_ZMQ_LIB = -lzmq
```

Comment:
It bothers me that these take on different values (an explicit filename "libzmq.a" vs a linker request "-lzmq") depending on whether CHPL_ZMQ_HOME is defined or not, but considering what is supposed to happen here I'm not sure if there's an easy way around it.

Reply:
I could have two environment variables in that command line with different names (and only define one of them, depending on whether CHPL_ZMQ_HOME was set or not).  I prefer this way, though (since technically `-lzmq` and `/full/path/to/libzmq.a` are functionally identical)

Reply:
Fair enough.